lib/main.dart

```diff
@@ -3,235 +3,159 @@ import 'dart:async';
+                                    child: Row(children: <Widget>[
+                                      const LinkedinIcon(),
+                                      Text(
+                                        snapshot.data!.linkedin,
+                                        style: Styles.contactStyle,
+                                        textAlign: TextAlign.center,
+                                      ),
+                                    ]),
+                                  ),
+                                  const SizedBox(
+                                    height: 40,
+                                  ),
+                                  Column(
+                                    children: <Widget>[
+                                      Center(
+                                        child: IconButton(
+                                          icon: Bounce(
+                                            child: const LikeIcon(),
+                                            infinite: true,
+                                          ),
+                                          onPressed: incrementCounter,
+                                          iconSize: 50,
+                                        ),
+                                      ),
+                                      const Center(
+                                          child: Text(
+                                        'Recomendar',
+                                        style: TextStyle(
+                                          fontSize: 24,
+                                          color: Colors.black,
+                                        ),
+                                      ))
+                                    ],
+                                  )
+                                ]);
                           }
-                          else if (snapshot.hasError) {
-                            return Text('${snapshot.error}');
-                          }
-                          return const CircularProgressIndicator();
-                        }),
-                  ]),
-            ),
-              SizedBox(height: 40,),
-              Column(
-                children: <Widget> [
-                  Center(
-                    child: IconButton(
-                      icon: Bounce(child: ImageIcon(AssetImage('assets/images/like.png'),),
-                        infinite: true,
-                    ),
-                      onPressed: () { incrementCounter(); },
-                      iconSize: 50,
-                    ),
-                  ),
-                  Center(
-                    child: Text('Recomendar', style: TextStyle(
-                      fontSize: 24,
-                      color: Colors.black,
-                    ),)
-                  )
-                ],
-              )
-          ])
-      ))));
+                        })))));
```

Comment:
to split this, try adding commas between parenthesis, the IDE should be showing you several name of widgets in a line